src/lib/patternfly-4-cockpit.scss

```diff
@@ -0,0 +1,14 @@
+/* Here we need to import patternfly-base.scss but the fonts file from PF4 is containing
+ * urls to font formats we are not interested in including.
+ * @import '@patternfly/patternfly/patternfly-base.scss';
+ */
+$pf-global--font-path: '/static/fonts';
+$pf-global--fonticon-path: '/base1/fonts';
+$pf-global--disable-fontawesome: true !default; // Disable Font Awesome 5 Free
+@import "../../node_modules/@patternfly/patternfly/sass-utilities/all";
+@import "../../node_modules/@patternfly/patternfly/variables";
+@import "../../node_modules/@patternfly/patternfly/base";
+@import "../../node_modules/@patternfly/patternfly/themes";
+@import "../../node_modules/@patternfly/patternfly/icons";
+
+@import "./fonts";
```

Comment:
That looks inconsistent -- file is "_fonts.scss", and it tries to import "fonts"?

Reply:
See above comment.